tmk_core/common/command.h

```diff
@@ -15,8 +15,7 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 
-#ifndef COMMAND_H
-#define COMMAND
+#pragma once
```

Comment:
I suggest changing the offending line to `#define COMMAND_H` instead. The rest of `tmk_core` code uses traditional header guards, and it'd be nice to remain consistent.

I'm all for `#pragma once`, so maybe we can make a separate PR that changes all existing traditional header guards to `#pragma once`? I just wouldn't do it here.

Reply:
Actually, I'm fine with `#pragma once`. You're right, we don't have to switch everything at once. Baby steps :smile: